pandas/core/series.py

```diff
@@ -1002,36 +1002,88 @@ def _set_value(self, label, value, takeable=False):
 
     def reset_index(self, level=None, drop=False, name=None, inplace=False):
         """
-        Analogous to the :meth:`pandas.DataFrame.reset_index` function, see
-        docstring there.
+        Generate a new DataFrame with columns containing the data and 
+        the index of the original Series, or, if drop is True, 
+        a Series with numeric index.
+        
+        For an Index Series, the results is a two-column DataFrame with 
```

Comment:
Index Series doesn't make sense, more like: For a Series with a single level index (Index)